### ISSUE TITLE
Failing tests: bcf validation

### ISSUE THREAD

**Issue by norbusan** [plk/biber#143]
I am building biber on Debian and see the following test errors:

`````
ok 41 - Validation of tdata/uniqueness2.bcf
tdata/uniqueness3.bcf:0: Relax-NG validity error : Error validating value 
tdata/uniqueness3.bcf:0: Relax-NG validity error : Element key failed to validate content
tdata/uniqueness3.bcf:0: Relax-NG validity error : Error validating value 
tdata/uniqueness3.bcf:0: Relax-NG validity error : Element key failed to validate content
tdata/uniqueness3.bcf:0: Relax-NG validity error : Error validating value 
tdata/uniqueness3.bcf:0: Relax-NG validity error : Element key failed to validate content
# Looks like you planned 45 tests but ran 41.
# Looks like your test exited with 255 just after 41.
Dubious, test returned 255 (wstat 65280, 0xff00)
Failed 4/45 subtests 
`````

This is with libxml-libxml-perl 2.0128+dfsg providing XML/LibXML, perl 5.24.1

Thanks

Norbert

**Comment by plk** [comment]
Hmm, I have a feeling that was a broken test file in an innitial 2.6 release which was corrected quickly. You can ignore that or pull the code from the v2.6 tag in git which should be ok.

**Comment by norbusan** [comment]
Hi Phil,
no, that didn't work. There is only one problematic file and it is uniqueness3.bcf, while the version 2.6 from git changes many other bcf files, but not uniqueness3.bcf.

I think I will simply remove the failing file and adjust the test number.

Thanks

Norbert

**Comment by plk** [comment]
Ok - it's an inconsequential test for users anyway. It is certainly fixed in 2.7.

**Comment by norbusan** [comment]
Ok, thanks.
